lib/ui/screen/home.dart

```diff
@@ -97,6 +99,13 @@ class HomePage extends ConsumerWidget {
                 },
                 child: Text(localization.twitter),
               ),
+              OutlinedButton(
+                onPressed: () async {
+                  final uri = Uri.parse('https://medium.com/flutterkaigi');
+                  await launchInExternalApp(uri);
+                },
+                child: Text(localization.medium),
+              ),
```

Comment:
Swapped the positions of the Twitter and GitHub buttons to align with the design.

Previous: 
<img src="https://github.com/FlutterKaigi/conference-app-2023/assets/1630075/8064a56a-a6d7-4c04-a90f-b5072c8fb49a" width="240" />


Figma: 

<img src="https://github.com/FlutterKaigi/conference-app-2023/assets/1630075/60fb7a5e-4da5-45e7-9918-ba292c3a6346" width="240" />